### DISCUSSION TITLE
No Wifi-Adapter shown in Ubuntu 24.04 LTS on Surface 3

### DISCUSSION THREAD

**Discussion by 10vorne** [linux-surface/linux-surface#1489] [Q&A]
I installed the latest Ubuntu version (Ubuntu 24.04 LTS) on a Surface 3 according to the instructions here. Everything went smoothly so far, I had a problem with Secure Boot, but now everything works. However, I don't have WiFi. Ubuntu tells me I don't have a Wifi adapter installed. Bluetooth is shown to me as available. Do I have to reinstall something?

**Comment by hirez1972** [comment]
I've just installed Ubuntu 24.04 on my Surface 3 (non pro) and the Wifi works, but it was initially flakey. It was repeatedly disappearing mid browsing. Only a reboot would bring it back. It seems to have stabilised now, although I have had a few system freezes. I didn't do anything in particular that made it calm down, so I can only wish you good luck!

**Reply by 10vorne** [reply]
I had this system freezes too. I haven't used the system in the last few weeks, I'll boot it up and check for updates and test it in terms of wifi and the system freezes. I will report.
